hs_app_netCDF/models.py

```diff
@@ -571,9 +571,7 @@ def get_update_netcdf_file_html_form(self):
         form_action = "/hsapi/_internal/netcdf_update/{}/".\
             format(self.resource.short_id)
         style = "display:none;"
-        if self.is_dirty:
```

Comment:
Could you explain to me why this check is going away?

Reply:
It was used to populate a context variable that is no longer needed and was removed in this branch.

Reply:
Correction. It was used to render inline CSS which was replaced by a utility class.